### DISCUSSION TITLE
[question] How do I navigate through the css to find how to resize the logo?

### DISCUSSION THREAD

**Discussion by moeraccoon** [searxng/searxng#2506] [Q&A] [answered]
I could find where to edit the file to replace the logo with a custom one, but the css file (.../themes/simple/css/searxng.min.css) is a one liner and moving around it takes a very long time, especially not knowing what I'm exactly looking for.

What should I find and edit to make the logo larger? https://search.raccoonden.moe/
As you can see at the moment the logo is very small.
While I'm at it, is it possible to fix the file to make it readable for human beings without breaking it?

**Comment by return42** [comment, marked as answer]
> What should I find and edit to make the logo larger? https://search.raccoonden.moe/

Customizing of the simple theme is not supported by the SearXNG development.

To answer your question: these files are a build product / the development cylcle is described in the developer docs https://docs.searxng.org/dev/quickstart.html
